src/sweeper/sweepers/duplicates.py

```diff
@@ -7,52 +7,63 @@
 
 import arcpy
 import re
+import os
 from xxhash import xxh64
 
-
 class DuplicateTest(object):
-    def __init__(self):
+    def __init__(self, workspace, table_name):
+        self.report = {}
+        self.workspace = workspace
+        self.table_name = table_name
+
+
+    def sweep(self):
 
+        digest_dict = {}
 
-    def sweep(self, fc):
+        dig_trim = re.compile(r'(\d+\.\d{2})(\d+)')
 
-        digestDict = {}
-        duplicateDict = {}
-        dupePrintLst = []
-        digTrim = re.compile(r'(\d+\.\d{2})(\d+)')
+        arcpy.env.workspace = self.workspace
 
-        shps = ['Shape', 'SHAPE']
+        def is_skip_field(field):
+            return 'SHAPE' in field.upper() or field.upper() in ['OBJECTID', 'FID', 'GLOBAL_ID', 'GLOBALID']
```

Comment:
this should be using the describe properties for the field names instead of a list of fields

Reply:
I was going off what Scott did

Reply:
I/we've grown up a little since that code was written. :)

Reply:
We know where zack will be when scott jumps off the bridge :)

Reply:
Hey, it's zaque to you. I'll get it fixed in the next day or so.